packages/core/core/src/loadParcelConfig.js

```diff
@@ -79,18 +80,90 @@ export async function readAndProcess(
   return processConfig(config, configPath, options);
 }
 
+function relatifyPipeline(
```

Comment:
I also had trouble with naming things in this file 🙃

Here are some suggestions, let me know what you think:
```
processConfig -> processConfigChain
preprocessConfig -> processConfig
readAndProcessConfig -> readAndProcessConfigChain (we could maybe roll readAndProcessConfigChain into resolveAndProcessConfigChain because I don't see anywhere we are not resolving before reading and processing, not super important though)
ParcelConfigFile -> RawParcelConfig
PreProcessedParcelConfig -> ProcessedParcelConfig
relatifyPipeline -> processPipeline
relatifyMap -> processMap
```

Reply:
Definitely better naming than what I used, changed it :)